Sources/MapboxDirections/RouteRefreshResponse.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+
+public struct RouteRefreshResponse {
```

Comment:
I’m not sure it’s meaningful for RouteRefreshResponse to be public while its main data is locked away in a private `legAnnotations`. Until client code needs to use RouteRefreshResponse directly, let’s make this struct private.

Reply:
Never mind, this struct is used in RouteRefreshCompletionHandler, which is public. Not sure how I missed that earlier.

Reply:
I ended up making the whole shebang public so that client code can take responsibility for merging.